### ISSUE TITLE
Banning for writing any comment

### ISSUE THREAD

**Issue by koolked** [Trumani/stop-spammers#190]
Hello,

First of all, thank you for the plugin.

I have noticed that any comment written in the WordPress comment box is banning users. This means a loss of a lot of traffic. For no reason whatsoever.

I have checked the plugin options and I don’t see anything to make this happen. According to the log, the reason is ‘allow admin-ajax.php’. Which makes no sense, since it is normal that when a person writes a comment admin-ajax is activated.

**Comment by bhadaway** [comment]
Is there anything unique about the implementation of these comments, meaning are they standard WordPress comments or are they third-party comments (Disqus, bbPress, WooCommerce, etc.)?

**Comment by koolked** [comment]
Contrary to expectations, the problem occurs with Wordpress comments. 

I have tried with wpdiscuz and it doesn't happen. However when I include a normal comment widget, in this case through Elementor, any comment is an instant ban.

**Comment by bhadaway** [comment]
It's likely it's the Elementor part that's not compatible. Try:

_Stop Spammers > Protection Options > Toggle on the option for “Only Use the Plugin for Standard WordPress Forms” > Save_
